### PR TITLE
Minor enhancements on phi and theta retrieval in python API

### PR DESCRIPTION
**Pull request by AnyaP**
1. Add model_name to get_phi arguments to be able to extract not only phi, but any word-topic matrix (e.g. nwt counters) in a similar way.
2. Add flag use_doc_titles to fit_transform arguments to name dataframe columns by document titles from Vowpal Wabbit input file.
3. Also I suggest that fit_transform should be renamed to get_theta, because its code is analogous to get_phi. Moreover, it never performs any "fit" (either it returns cached theta, or raises an error), so an analogy with sklearn is misleading.

### CODE REVIEW

File: python/artm/model.py [hunk 1]
```diff
@@ -638,8 +640,11 @@ def get_phi(self, topic_names=None, class_ids=None):
         if not self._initialized:
             raise RuntimeError('Model does not exist yet. Use ARTM.initialize()/ARTM.fit_*()')
 
-        phi_info = self.master.get_phi_info(model=self.model_pwt)
-        nd_array = self.master.get_phi_matrix(model=self.model_pwt,
+        if model_name is None:
+            model_name = self.model_pwt
```
Comment (sashafrey) [inline python/artm/model.py]:
```
if model_name is None:
  model_name = self.model_pwt
```
I'm not sure if in python this works as you expect it... From what I understand this this code will actually change the default value for `model_name` argument that will affect next calls to `get_phi`. @MelLain , am I correct? How would you suggest to change this?

File: python/artm/model.py [hunk 2]
```diff
@@ -653,7 +658,7 @@ def get_phi(self, topic_names=None, class_ids=None):
 
         return phi_data_frame
 
-    def fit_transform(self, topic_names=None, remove_theta=False):
+    def get_theta(self, topic_names=None, remove_theta=False, use_doc_titles=False):
```
Comment (sashafrey) [inline python/artm/model.py]: I'd keep old method too for now (ideally tagged with `obsolete`, but I don't know how to do this in python)
```
def fit_transform(self, topic_names=None, remove_theta=False):
  return self.get_theta(topic_names, remove_theta);
```

File: python/artm/model.py (hunk 1 above)
Comment (MelLain) [inline python/artm/model.py]: Don't change model_name, it's an argument passed by reference. Use smth like this

final_model_name = model_name if model_name is not None else self.model_pwt
phi_info = self.master.get_phi_info(model=final_model_name)
nd_array = self.master.get_phi_matrix(model=final_model_name)

File: python/artm/model.py [hunk 3]
```diff
@@ -662,6 +667,8 @@ def fit_transform(self, topic_names=None, remove_theta=False):
           default=None (means all topics)
           remove_theta (bool): flag indicates save or remove Theta from model
           after extraction, default=False
+          use_doc_titles (bool): name columns of theta dataframe by document titles,
```
Comment (sashafrey) [inline python/artm/model.py]: Thank you for fixing #308 , this issue had been opened for way too long :)
Have you consider other ways of customizing the behavior? The issue with adding a Boolean flag to `fit_transform` method is that 
* that this flag needs to be in both methods (`get_theta` and `transform`). 
* most likely users want to configure the behavior once and for all calls. 
I would consider adding a string property on `ARTM` class, named for example `ThetaMatrixColumnsType`. Expected values should be `id` and `title`.

**Comment by sashafrey** [comment]
:+1:

**Pull request merged**